lib/azure/armrest/model/base_model.rb

```diff
@@ -263,3 +263,4 @@ class Snapshot < BaseModel; end
 end
 
 require_relative 'storage_account'
+require_relative 'virtual_machine'
```

Comment:
How could the gem work before without this line? From the your code changes I cannot see the additional need. Was it the spec test?